after_deployment_tasks.md

```diff
@@ -13,6 +13,14 @@ Check that you now
 
 * see "1+2=3" in each app.
 * have been redirected to https
+
+Setup cloudflare: https://www.cloudflare.com/cloudflare-settings?z=renuoapp.ch
+
+* Now open
+  * < application>-master.renuoapp.ch/home/check
+  * < application>-develop.renuoapp.ch/home/check
+  * < applicaiton>-testing.renuoapp.ch/home/check
```

Comment:
thx, fixed! but we need to use the same name everywhere in the future, so we can automatically replace them in all template files when copying (see also comment in the book where the template files should be copied)